R/print.data.table.R

```diff
@@ -86,6 +86,14 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
     if (show.indices) toprint = cbind(toprint, index_dt)
   }
   require_bit64_if_needed(x)
+  if (!inherits(toprint, "data.frame")) {
+    class(toprint) <- c("data.table", "data.frame")
```

Comment:
Could you please explain why this is needed?

Reply:
This error is signalled by `colnames(toprint)`. Since the `col_names` variables isn't used anywhere, it's better to remove both it and the `inherits(toprint, ...)` check.